### ISSUE TITLE
Credential plugin cache directory is created in cwd

### ISSUE THREAD

**Issue by twiebe** [gridscale/gscloud#8]
We're creating the gscloud cache directory in the current working dir at the moment:
```
$ pwd
/Users/thomas/tmp/gscloud
$ ls -la
total 0
drwxr-xr-x   2 thomas  staff    64B Jan 13 11:57 .
drwxr-xr-x  72 thomas  staff   2.3K Jan 13 11:57 ..
$ kubectl get nodes
NAME           STATUS   ROLES    AGE     VERSION
node-pool0-0   Ready    <none>   4d20h   v1.16.4
node-pool0-1   Ready    <none>   4d14h   v1.16.4
$ ls -la
total 0
drwxr-xr-x   3 thomas  staff    96B Jan 13 11:57 .
drwxr-xr-x  72 thomas  staff   2.3K Jan 13 11:57 ..
drwx------   3 thomas  staff    96B Jan 13 11:57 cache
$ find cache -ls
45820327        0 drwx------    3 thomas           staff                  96 Jan 13 11:57 cache
45820328        0 drwx------    3 thomas           staff                  96 Jan 13 11:57 cache/exec-credential
45820329       16 -rw-------    1 thomas           staff                5442 Jan 13 11:57 cache/exec-credential/1358261b-ca50-4c35-a2b1-xxx.json
```
Let's move that to an os-specific, standardized location. The freedesktop spec f.e. defines $XDG_CACHE_HOME for this (https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html). So on linux it would be: `~/.cache/gscloud`

@bkircher do you have any ideas on how to make this as compliant as possible. Is there some library we can use?

**Comment by itakouna** [comment]
@twiebe we used the standardized location for gscloud config. I'll move the gscloud cache to be on the standardized location as well. would this the solution that you are looking for?

**Comment by itakouna** [comment]
solved by #9
